src/routes/locking/locking.controller.ts

```diff
@@ -42,6 +43,25 @@ export class LockingController {
     return this.lockingService.getCampaigns({ routeUrl, paginationData });
   }
 
+  @ApiOkResponse({ type: HolderPage })
+  @ApiQuery({
+    name: 'cursor',
+    required: false,
+    type: String,
+  })
+  @Get('/campaigns/:campaignId/leaderboard')
+  async getLeaderBoardByCampaignId(
```

Comment:
As before regarding "leaderboard":

```suggestion
  async getLeaderboardByCampaignId(
```

Reply:
I'm resolving this one as this got renamed to `getCampaignLeaderboard`. Lmk if you have other suggestion of course!